deploy/olm-catalog/akka-cluster-operator/1.0.0/akka-cluster-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,320 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator.v0.2.3
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'false'
+    containerImage: 'lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:v0.2.2'
```

Comment:
version 1.0.0?

Reply:
Yes, if you push a 1.0.0 tag to [bintray](https://bintray.com/lightbend/registry/lightbend%3Aakkacluster-operator).